modules/Microsoft.Network/azureFirewalls/deploy.bicep

```diff
@@ -117,7 +117,7 @@ param diagnosticMetricsToEnable array = [
 @description('Optional. The name of the diagnostic setting, if deployed.')
 param diagnosticSettingsName string = '${name}-diagnosticSettings'
 
-var additionalPublicIpConfigurations_var = [for ipConfiguration in additionalPublicIpConfigurations: {
+var additionalPublicIpConfigurationsVar = [for ipConfiguration in additionalPublicIpConfigurations: {
```

Comment:
Hi @gbeaud, this might already be covered by a broader update in PR #2329. Would you mind closing this PR and wait for the planned #2329 to be merged?

Reply:
The PR was merged. @gbeaud please fetch upstream in your fork and let's check if the issue was already resolved. If not, the PR should automatically be updated and can go in :)